auxiliary_tools/cdat_regression_testing/660-cosp-histogram/660_cosp_histogram.py

```diff
@@ -0,0 +1,42 @@
+# %%
```

Comment:
TODO: Delete this file because it is replaced by `660_cosp_histogram_run_script.py`.